src/app/store/selectors.js

```diff
@@ -25,6 +25,16 @@ export function isTopLevel(state) {
 	return state.app.isTopLevel;
 }
 
+export function isOnECommercePlan(state) {
+  return ["wc_standard", "wc_premium"].includes(
+    state.app.customer.plan_subtype
+  );
+}
+
+export function isNewEcommerceUser(state) {
+	return new Date(state.app.customer.signup_date) >= new Date('2022-08-18T17:00:00.000Z');
```

Comment:
@aulisius Per a meeting with Jocelyn just now, we've picked a date/time of August 18 at 10AM Mountain Time I pushed here now. This is something we'll be addressing in Onboarding as well.
![Screenshot 2022-08-03 @ 03 02 29 PM](https://user-images.githubusercontent.com/9565066/182720861-2b3d2e92-cc3a-4a92-b4df-675f160929d5.jpg)

Reply:
Understood.